src/pages/MerchantGiftCards/MerchantGiftCards.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  getMerchantGiftCards,
+  getSeller,
+} from '../../utilities/api/interactionManager';
+import Loader from '../../components/Loader/Loader';
+import styles from './styles.module.scss';
+import ErrorPage from '../../components/404Page';
+const FilterableTable = require('react-filterable-table');
+
+const MerchantGiftCards = () => {
+  const params = useHistory();
+  let urlParams = params.location.pathname.match(/\/[^/]+/g) as string[];
+  urlParams = urlParams.map((param) => param.replace('/', ''));
+  const metadata = {
+    sellerId: urlParams[0],
+    secretId: urlParams[2],
+  };
+
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [giftCards, setGiftCards] = useState<any | null>();
+  const [seller, setSeller] = useState<any | null>();
+
+  const renderAmount = (props) => {
```

Comment:
consider declaring a type for props? Same for the other anonymous functions

Reply:
Yeah, these have an expected format from the underlying library... where I wish they were typed, but I can at least define them here
https://github.com/ianwitherow/react-filterable-table